octopoes/octopoes/models/ooi/email_security.py

```diff
@@ -122,6 +182,8 @@ def format_reference_human_readable(cls, reference: Reference) -> str:
 
 
 class DKIMExists(OOI):
+    """Represents the the DKIM exists object if a DKIM record was identified."""
```

Comment:
1. There is a double "the" in this sentence
2. This object represents whether a DKIM can exist by checking the DNS response of _domainkey.hostname